automated-comments/go/hamming/invert_if_for_happy_path_on_left.md

```diff
@@ -0,0 +1,6 @@
+To simplify, invert the `if` condition handling the error case, 
+then get rid of the `else` branch by returning within the `if`.
+A Go proverb says "[The happy path is left-aligned](https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88)",
+meaning the normal flow of control follows the left-hand margin, 
+and only exceptional or error cases are indented.
+
```

Comment:
The other files don't have a trailing empty line. I doubt it's a problem, but if we want to be consistent we could ✂️ it.